SpatialGDK/Documentation/content/singleton-actors.md

```diff
@@ -1,30 +1,25 @@
-<%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
-
 # Singleton Actors
 
 Singleton Actors allow a single source of truth for both operations and data across a multi-server simulation. They are server-side authoritative [Actors](https://docs.unrealengine.com/en-us/Programming/UnrealArchitecture/Actors) that are restricted to one instantiation on SpatialOS.
 
 There are two kinds of Singleton Actors:
 
 * **Public Singleton Actors** - Singleton Actors which are replicated to server-workers and client-workers. `GameState` is a Public Singleton Actor.
-* **Private Singletons** - Singleton Actors which are replicated to server-workers, but not accessible to client-workers. `GameMode` is a Private Singleton Actor.
+* **Private Singleton Actors** - Singleton Actors which are replicated to server-workers, but not accessible to client-workers. `GameMode` is a Private Singleton Actor.
```

Comment:
Nice.

Reply:
Link to glossary term here (I have put this in backticks so you can see the link but we don't need the backticks in the text). I think in this instance, we should put the links in for both entries - even though they are the same link. (Normally, you only do it once but it feels more right to do it each time here??!?)
`....which are replicated to [server-workers and client-workers]({{urlRoot}/content/glossary#workers) .`

AND

`replicated to [server-workers]({{urlRoot}/content/glossary#workers), but not accessible to [client-workers]({{urlRoot}/content/glossary#workers).